### PR TITLE
Feature/display articles

### PR DESCRIPTION
**Pull request by eliotmess**
changes:
    •  displaying list of articles, styling
to be changed next: 
    • dynamic routing to each article's page

### CODE REVIEW

File: src/app/pages/MainArticlesScreen/components/ArticleComponent/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { injectIntl } from 'react-intl';
+
+import ArticleComponent from './Article.component';
+
+export const Article = compose(
+  injectIntl,
+  connect(),
```
Comment (lukaszblasz) [inline src/app/pages/MainArticlesScreen/components/ArticleComponent/index.ts]: empty connect?

Comment (eliotmess) [reply src/app/pages/MainArticlesScreen/components/ArticleComponent/index.ts]: I left it based on our create-web-app (was it left to make it easier to extend later?). 
Should I delete it?

File: src/app/pages/MainArticlesScreen/components/ArticleComponent/Article.component.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+class ArticleComponent extends PureComponent<MainScreenProps> {
+  public render() {
+    const { intl, article } = this.props;
+    return (
+      <ArticleWrapper>
+        <ArticleImage
+          resizeMode="cover"
+          source={{
+            uri: article.urlToImage ? article.urlToImage : fallbackImage,
```
Comment (adamkosmala) [inline src/app/pages/MainArticlesScreen/components/ArticleComponent/Article.component.tsx]: what about `article.urlToImage ?? fallbackImage` ?

File: src/app/pages/MainArticlesScreen/components/ArticleComponent/Article.component.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          }}
+        />
+        <ArticleMetaDataWrapper>
+          <ArticleMetaTitle>
+            {dayjs(article.publishedAt).format('YYYY, MMM DD ')}
+          </ArticleMetaTitle>
+          {article.author ? (
```
Comment (adamkosmala) [inline src/app/pages/MainArticlesScreen/components/ArticleComponent/Article.component.tsx]: what about moving the ternary condition inside `ArticleMetaTitle`?

File: src/app/pages/MainArticlesScreen/MainArticlesScreen.tsx [hunk 4]
```diff
@@ -36,7 +35,7 @@ class MainScreen extends PureComponent<MainScreenProps> {
     });
   };
 
-  private handleFilterPicker = (value: string) => {
+  private handleFilterPicker = ({ value }: Pick<PickerData, 'value'>) => {
```
Comment (adamkosmala) [inline src/app/pages/MainArticlesScreen/MainArticlesScreen.tsx]: 👍

File: src/app/pages/MainArticlesScreen/components/ArticleComponent/Article.component.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+interface OwnProps {
+  article: ArticleData;
+}
+
+type MainScreenProps = WrappedComponentProps & OwnProps;
+
+class ArticleComponent extends PureComponent<MainScreenProps> {
+  private goToOwnPage() {
+    console.log('go to');
```
Comment (adamkosmala) [inline src/app/pages/MainArticlesScreen/components/ArticleComponent/Article.component.tsx]: ?

Comment (eliotmess) [reply src/app/pages/MainArticlesScreen/components/ArticleComponent/Article.component.tsx]: gonna be implemented with next PR

**Pull request merged**